crdt/rlist.go

```diff
@@ -926,20 +953,109 @@ func (l *RList) DeleteCharAt(i int) error {
 	return l.DeleteChar()
 }
 
+// +-----------------------------------+
+// | Operations - Insert str container |
+// +-----------------------------------+
+
+//Inserts a string container as a child of the root atom.
+type InsertStr struct{}
+
+func (v InsertStr) AtomPriority() int { return 30 }
+func (v InsertStr) MarshalJSON() ([]byte, error) {
+	return json.Marshal("insert str container")
+}
+
+func (v InsertStr) String() string { return "STR: " }
+
+func (v InsertStr) ValidateChild(child AtomValue) error {
+	switch child.(type) {
+	case InsertChar:
```

Comment:
You can put multiple types in here, like

    case InsertChar, Delete:

Note: This wouldn't be possible if you were assigning the converted type to a variable, like

    switch x := child.(type) {
      case InsertChar, Delete:
        // What's x's type now?